libs/clients/adr-and-machine-license/README.md

```diff
@@ -0,0 +1,7 @@
+# Clients ADR and Machine License
```

Comment:
Should make title and navigation title consistent to other client libs.

Reply:
Will do (later)